modules/stochastic_tools/src/distributions/BoostNormal.C

```diff
@@ -0,0 +1,37 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "BoostNormal.h"
+#include "Normal.h"
+
+#include "libmesh/auto_ptr.h"
+
+// registerMooseObjectAliased("StochasticToolsApp", BoostNormalDistribution, "BoostNormal");
+registerMooseObjectReplaced("StochasticToolsApp", BoostNormal, "07/01/2020 00:00", Normal);
+// registerMooseObjectReplaced("StochasticToolsApp",
+//                            BoostNormal,
+//                            "07/01/2020 00:00",
+//                            Normal);
```

Comment:
Left over comments?

Reply:
Whoops!  I should have all these cleaned up now.